test/test_unit.jl

```diff
@@ -1709,6 +1713,42 @@ end
                     0.13942836392866081 0.3605716360713392], atol = 1e-13)
 end
 
+@testset "PERK Single p3 Constructors" begin
+    path_coeff_file = mktempdir()
+    Trixi.download("https://gist.githubusercontent.com/warisa-r/0796db36abcd5abe735ac7eebf41b973/raw/32889062fd5dcf7f450748f4f5f0797c8155a18d/a_8_8.txt",
+                   joinpath(path_coeff_file, "a_8.txt"))
+
+    ode_algorithm = Trixi.PairedExplicitRK3(8, path_coeff_file, 42) # dummy optimal time step (dt_opt plays no role in determining `a_matrix`)
```

Comment:
This is probably not directly related to this PR, but rather to the previous one, which implemented the `calculate_cfl` function for `PairedExplicitRK2`.
Do I understand it correctly that the `dt_opt` is only needed to be passed to this constructor if you want to use the `StepsizeCallback` with a CFL number compute with `calculate_cfl`? If that is the case, could we provide a default value like `nothing` for `dt_opt`? I'm asking because now, as I understand correctly, it is not possible to use `PairedExplicitRK{2,3}` if you 1) have a path to the coefficients of the Butcher tableau, 2) don't have an optimal dt, and 3) don't want to use a `StepsizeCallback`. However, in principle it should be possible to use `PairedExplicitRK{2,3}` also in that case, right?

Reply:
You are right. The user should be able to use the integrator in the case you mentioned as well. This fix is implemented for PERK2 first in a different PR along with completing the docstring in PERK2 in this #2123 .